### PR TITLE
ASAP is a massively parallel molecular dynamics simulator for ASE.

### PR DESCRIPTION
**Pull request by schiotz**
(created using `eb --new-pr`)

### CODE REVIEW

File: easybuild/easyconfigs/a/ASAP3/ASAP3-3.10.7-foss-2016b-Python-3.5.2.eb [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+
+toolchain = {'name': 'foss', 'version': '2016b'}
+
+source_urls = [PYPI_LOWER_SOURCE]
+sources = [SOURCELOWER_TAR_GZ]
+checksums = ['20d721f3eeac6982132eda91d9309baaea03f6ddeab418a0744879daf353c773']
+
+python = 'Python'
+pythonver = '3.5.2'
+versionsuffix = '-%s-%s' % (python, pythonver)
```
Comment (boegel) [inline easybuild/easyconfigs/a/ASAP3/ASAP3-3.10.7-foss-2016b-Python-3.5.2.eb]: please drop the `python` & `pythonver` variables, and use `-Python-%(pyver)s`  as `versionsuffix` (and move it up below `version =`

File: easybuild/easyconfigs/a/ASAP3/ASAP3-3.10.7-foss-2016b-Python-3.5.2.eb [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+
+ase = 'ASE'
+asever = '3.15.0'
```
Comment (boegel) [inline easybuild/easyconfigs/a/ASAP3/ASAP3-3.10.7-foss-2016b-Python-3.5.2.eb]: no need for these either, only used in one place, just include the values directly in `dependencies` like you should for `Python`

**Comment by schiotz** [comment]
@boegel Sorry about the style problems, I should have known.  I have fixed it.

I also abandon the iomkl toolchain version for now, as building with iomkl-2017b and depending on a Python built with foss-2017b cause a dependency conflict that Travis picks up.  It works, however, fine on our cluster.

I will look into providing an Intel compiled version later.  It does give a factor two in performance, so it is worth it. :)

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in this PR)
node2639.swalot.os - Linux centos linux 7.4.1708, Intel(R) Xeon(R) CPU E5-2660 v3 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/6d50e96a64b8686b71dbef59b3fd6321 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in this PR)
node2639.swalot.os - Linux centos linux 7.4.1708, Intel(R) Xeon(R) CPU E5-2660 v3 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/88239cfc2e3c7650891198ec9731cb58 for a full test report.

**Comment by boegel** [comment]
Going in, thanks @schiotz!

**Pull request merged**